src/EnlistmentInfo.targets

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Well, I am one of the contributors to it, so I personally think it's awesome, but I am biased :smile: